SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -219,9 +229,10 @@ public static void Main(string[] args)
                 });
 
                 var linuxSimulatedPlayerPath = Path.Combine(stagingDir, "LinuxNoEditor");
-                File.WriteAllText(Path.Combine(linuxSimulatedPlayerPath, "StartWorker.sh"), SimulatedPlayerWorkerShellScript.Replace("\r\n", "\n"), new UTF8Encoding(false));
+                File.WriteAllText(Path.Combine(linuxSimulatedPlayerPath, "StartSimulatedClient.sh"), string.Format(SimulatedPlayerWorkerShellScript, baseGameName).Replace("\r\n", "\n"), new UTF8Encoding(false));
+                File.WriteAllText(Path.Combine(linuxSimulatedPlayerPath, "StartCoordinator.sh"), string.Format(SimulatedPlayerCoordinatorShellScript, baseGameName).Replace("\r\n", "\n"), new UTF8Encoding(false));
 
-                var workerCoordinatorPath = Path.GetFullPath(Path.Combine("../spatial", "build", "dependencies", "WorkerCoordinator"));
+                var workerCoordinatorPath = Path.GetFullPath(Path.Combine("Plugins", "UnrealGDK", "SpatialGDK", "Binaries", "ThirdParty", "Improbable", "Programs", "WorkerCoordinator"));
```

Comment:
Will this work if the plugin is an engine plugin also?

Reply:
Made a suggestion on how to fix this. Since we're already inside the plugin when running this code we can easily access the `"SpatialGDK", "Binaries", "ThirdParty", "Improbable", "Programs", "WorkerCoordinator"` path